subsys/bluetooth/mesh/Kconfig

```diff
@@ -109,6 +109,38 @@ config BT_MESH_PROXY_FILTER_SIZE
 
 endif
 
+if BT_MESH_USES_TFM_PSA
+
+choice TFM_PROFILE_TYPE
+	default TFM_PROFILE_TYPE_NOT_SET
+endchoice
+
+config PSA_CRYPTO_DRIVER_OBERON
+	default y
```

Comment:
Please remove this, it's default y already.
We will make crypto_driver selection automatic based on PSA_WANT symbols, so the protocol shouldn't have to worry about this.